### PR TITLE
Updated images for iMovo subs card launch

### PR DESCRIPTION
**Pull request by i-hardy**
## Why are you doing this?

Adding the updated images for both the subscription card and home delivery tabs on the Print Subscriptions landing page, behind the existing `useDigitalVoucher` switch

[**Trello Card**](https://trello.com/c/cb6a4ZM1)

## Changes

* Adding new images
* Adjustments to styling to accommodate the new images in accordance with the designs

## Did you enjoy your PR experience?

 - [ ] [PR experience rated](https://forms.gle/N6FsTGG8JQFGV4Ha9)

## Screenshots

(images are not shown below desktop breakpoint)

Screenshot 2020-10-05 at 16 38 47
Screenshot 2020-10-05 at 16 38 57

### CODE REVIEW

File: support-frontend/assets/helpers/theGrid.js [hunk 1]
```diff
@@ -108,7 +108,8 @@ export const imageCatalogue: {
   printFeaturePackshot: '017a2f5c27394635b53c414962bbb775ce9b131d/5_39_1572_861',
   printCampaign2020: 'e50943f03b58caf6fde78d81eb65c92c048701c9/1844_0_1674_1006',
   printCampaignHero: 'fff86e98dbe83b36892baf7942f30fd3bcbcaee6/286_215_714_285',
-  printCampaigniMovo: 'f257e33b922f09ca180cc9205e0c766c2c2d529c/28_0_694_694',
+  printCampaigniMovo: '3cca8e05f3253f5a14329429ec3f6f139f84929a/0_21_562_337',
```
Comment (rupertbates) [inline support-frontend/assets/helpers/theGrid.js:111]: I think `printCampaignDigitalVoucher` would be a better name, we don't generally use iMovo in code I don't think

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (rupertbates) [inline yarn.lock:1]: I would revert this as nothing has actually changed

**Review by rupertbates — APPROVED** [review]
A couple of suggestions but nothing serious 👍

File: support-frontend/assets/pages/paper-subscription-landing/components/content/deliveryTab.jsx [hunk 3]
```diff
@@ -40,7 +40,7 @@ const ContentDeliveryFaqBlock = ({
   <Content
     border={paperHasDeliveryEnabled()}
     image={<GridImage
-      gridId="printCampaignHD"
+      gridId={useDigitalVoucher ? 'printCampaignHDiMovo' : 'printCampaignHD'}
```
Comment (lucymonie) [inline support-frontend/assets/pages/paper-subscription-landing/components/content/deliveryTab.jsx:43]: I think this is getting updated and isn't conditional on useDigitalVoucher (could be misunderstanding though!)

**Review by lucymonie — CHANGES_REQUESTED** [review]
Forgive me if my information is out of date, but I think the images for the order summary in the checkout are also meant to be updated in this PR.

**Review by lucymonie — APPROVED** [review]

**Pull request merged**